### PR TITLE
[WFLY-17349] Use a completely different entity with its own identity …

### PR DESCRIPTION
**Pull request by bstansberry**
…column name for webtxem tests""

https://issues.redhat.com/browse/WFLY-17349

See https://issues.redhat.com/browse/WFLY-17349?focusedId=23497895&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23497895 for details.

### CODE REVIEW

**Comment by bstansberry** [comment]
@scottmarlow Please review.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/webtxem/entity/WebJPAEntity.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
```
Comment (scottmarlow) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/webtxem/entity/WebJPAEntity.java:1]: @bstansberry I think this should be updated to:
```
/*
 * Copyright The WildFly Authors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Comment (bstansberry) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/webtxem/entity/WebJPAEntity.java:1]: @scottmarlow Absolutely! 

Ah, I see what happened. I started this branch by un-reverting the old fix commit, which restored the old LGPL webtxem/entityFlight.java file. I then I reworked that old file but left the LGPL header.

Comment (bstansberry) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/webtxem/entity/WebJPAEntity.java:1]: Fixed; thank Scott!

**Review by scottmarlow — APPROVED** [review]

**Pull request merged**